source/includes/azure/_subnets.md

```diff
@@ -73,9 +73,9 @@ Additional Attributes: Network filter included | &nbsp;
 curl -X GET \
    -H "MC-Api-Key: your_api_key" \
    "https://cloudmc_endpoint/v1/services/azure/example/subnets/subscriptions/:subscription/resourceGroups/:resourceGroup/providers/Microsoft.Network/virtualNetworks/:example-vnet/subnets/example-subnet"
```

Comment:
same comment as above except with example-vnet as well as resourcegroups and subscription

I guess they should be consistent in that the curl example does or does not have the `:var` syntax like the <code> part does ?

Reply:
Yes they should! I'll go over the docs again to see if I missed any.

Reply:
Nice catch, fixed these [here](https://github.com/cloudops/cloudmc-api-docs/pull/134/commits/996dc0a649c40dabc549d50add9dc3ba75bebec9#diff-1830ab3a9e537bffddd5e0c563f5846fL78)